skt/executable.py

```diff
@@ -241,7 +241,8 @@ def cmd_merge(cfg):
         with open(ktree.mergelog, 'r') as mergelog:
             for line in mergelog:
                 # Skip the useless part of the 'git am' output
-                if "The copy of the patch" in line:
+                if ("The copy of the patch" in line) \
+                        or ('see the failed patch' in line):
```

Comment:
This seems fine, but in the future an example output change in the changelog would be a helpful reference.  Otherwise I have to reproduce this failure manually and double check what you are filtering is indeed useless.  As a reviewer I need to double check the stated opinion sometimes. :-)

Reply:
Makes sense! I'll add examples of the new message found to the commit message

Reply:
@dzickusrh added the example and reasoning in the commit message, thanks for the idea!

Reply:
Looks great thanks!